source/whats-new.txt

```diff
@@ -32,4 +40,89 @@ What's New in 1.20
    Support for MongoDB Server v3.6 is removed in this release of the
    library.
 
-- Adds support for MongoDB Server v8.0.
\ No newline at end of file
+- Adds support for MongoDB Server v8.0.
```

Comment:
S: you can use the mdb-server constant here (applies to all mentions of MongoDB Server):
```suggestion
- Adds support for {+mdb-server+} v8.0.
```

Reply:
Why are constants helpful? Does it help with load speeds? Thanks!

Reply:
Sorry I missed this question, but I think it just helps with consistency (if a product name or syntax changes we wouldn't have to fix it multiple times, for example). Not sure about load speeds but that would make sense.